### DISCUSSION TITLE
Next Release

### DISCUSSION THREAD

**Discussion by jmsims2** [axios/axios#4595] [General]
Is there a planned release coming up? Looks like the latest version is missing the optional 3rd parameter type definition for `use` in interceptors. https://github.com/axios/axios/blob/master/index.d.ts#L210 

Just was wondering if we could expect that to get released soon.

Thanks for any insight!!

**Comment by jasonsaayman** [comment]
Working on it, I should have it out in the week

**Comment by jasonsaayman** [comment]
Completed
